vizro-core/docs/pages/explanation/why_vizro.md

```diff
@@ -0,0 +1,14 @@
+# Why Vizro
+
+In the context of comparison with other visualization packages or business intelligence tools, Vizro combines benefits of the speed and user-friendliness of low-code visual analytics tools with the flexibility and power of high-code tools - whilst providing inbuilt visual design, application architecture and coding standardization which also facilitates scaling and reusability.
+
+The simple and largely declarative configuration language follows a “grammar of dashboards” which makes the creation of complex dashboards more intuitive. This leverages modular components, empowering users to rapidly self-serve the assembly of customized dashboards with inbuilt visual styling in minutes, and reduces the amount of “glue code” required, whilst still retaining the ability to enhance functionality almost infinitely through customization and extensions, with standardization increasing the consistency across outputs.
+
+<figure markdown>
+  ![Image title](https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/code_dashboard.png)
+  <figcaption>Example of how Vizro's configuration language translates into an app</figcaption>
+</figure>
```

Comment:
Can you apply more indentation in the example? e.g.
```
page = vm.Page(
    title="Example Dashboard",
    components=[
        vm.Graph(figure=px.scatter(
            px.data.iris(),
            title="Scatter",
            x="sepal_length",
            y="petal_length",
            color="species",
        )),
        vm.Graph(figure=px.histogram(
            px.data.iris(),
            title="Histogram",
            x="sepal_length",
            color="species",
        ))
    ],
    controls=[
        vm.Filter(column="species"),
        vm.Filter(column="sepal_length"),
        vm.Filter(column="petal_length"),
    ]
)

Vizro().build(vm.Dashboard(pages=[page])).run()
```

Reply:
noted - this can be applied in a future PR when explanatory material and screenshots are next updated together